app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -683,11 +689,29 @@ public void checkObjectives(@NonNull Location location) {
      */
     public void initCircle() {
         circleManager.deleteAll();
+        circleHelper(256,game_center,circleRadius/5000.0);
```

Comment:
All of these values should be hardcoded, it is very hard to understand what all of them are otherwise.

Reply:
done